configs/xlml/tensorflow/solutionsteam_tf_nightly_supported_config.py

```diff
@@ -168,10 +170,12 @@ def get_tf_resnet_config(
   )
 
 
-def export_env_variable(is_pod: bool) -> str:
+def export_env_variable(is_pod: bool, is_pjrt: bool) -> str:
   """Export environment variables for training if any."""
-  return (
-      "export TPU_LOAD_LIBRARY=0"
-      if is_pod
-      else "export NEXT_PLUGGABLE_DEVICE_USE_C_API=true && export TF_PLUGGABLE_DEVICE_LIBRARY_PATH=/lib/libtpu.so"
-  )
+  if is_pod:
+    return "export TPU_LOAD_LIBRARY=0"
+  elif is_pjrt:
+    return "export NEXT_PLUGGABLE_DEVICE_USE_C_API=true && export TF_PLUGGABLE_DEVICE_LIBRARY_PATH=/lib/libtpu.so"
+  else:
+    # dummy command
+    return "echo"
```

Comment:
This gets prepended to the command right? So doesn't this mean the test won't actually run in this path?

Reply:
Thanks for the question Eric! The test will actually run in this path if it's single host and SE. The reason to put `echo` is to ensure `env_variable` has something to run between two `&` ([code](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/configs/xlml/tensorflow/solutionsteam_tf_nightly_supported_config.py#L55). Let me know your thoughts!